README.rst

```diff
@@ -24,6 +24,26 @@ Code Overview
 
 The program ``pipelines.py`` creates a graph using the Python package ``networkx``, and then uses the Ocean software tools to run the ``minimum_vertex_cover`` function from within the ``dwave_networkx`` package.
 
+Inside the Code: The Lagrange Parameter
+------ --- ----  --- -------- ---------
+Gamma, our Lagrange parameter, weights the constraints in the problem versus
+the energy term (objective). If the Lagrange parameter is too small, relative
+to the strength of the energy term, the constraints may be violated.
+
+A valid vertex cover is a subset of nodes in the graph such that every edge
+in the graph has at least one end point in the subset. Visual inspection shows
+that we will need either 1 or 2 in the solution; we will also need 6 or 7;
+and we will also need 4 or 5. We therefore know that a minimum vertex cover
+will have at least 3 nodes; it turns out that there must be four.
+
+With a Lagrange parameter that is too weak, the solutions are sets which are
+too small. In this code, the default of the Lagrange parameter is 2. 
```

Comment:
"In this code, the default" --> unclear if this refers to the dnx function or the demo code.

Reply:
thanks. Fixed.